lib/src/process_pool.dart

```diff
@@ -98,9 +164,65 @@ class WorkerJob {
   Exception? exception;
 
   @override
-  String toString() {
-    return command.join(' ');
+  void addToQueue(List<DependentJob> jobs) {
+    jobs.add(this);
   }
+
+  @override
+  String toString() => name;
+}
+
+/// A job that groups other jobs.
+///
+/// Each worker job, except the first, are made to be dependent upon the
```

Comment:
Maybe also mention the jobs are run in order.

Reply:
Done.